Makefile

```diff
@@ -416,7 +416,7 @@ build-test-system: compile-schemas update-version-file ## Build system tests doc
 
 .PHONY: package-wheel
 package-wheel: clean update-version-file ## Build python package wheel
-	python setup.py bdist_wheel
+	python -m build --wheel
```

Comment:
It's generally recommended to upload also the source distribution and not only the wheel:
https://packaging.python.org/en/latest/discussions/setup-py-deprecated/#id1

But maybe let's keep it for another time and document in Jira